src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/on.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'on(event, handler)'
+excerpt: 'Browser module: page.on method'
+---
+
+Allows the definition of an event handler function that will be executed every time the specified event is emitted.
+
+| Parameter       | Type   | Default | Description                                                                                                                                                                                                                           |
+|-----------------|--------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| event        | string  | `''`    |  Event to attach handler to. Currently, only `'console'` event is supported.                                                                                                               |
```

Comment:
```suggestion
| event        | string  | `''`    |  Event to attach the handler to. Currently, only the `'console'` event is supported.                                                                                                               |
```

Reply:
Fixed in [d49b29c](https://github.com/grafana/k6-docs/pull/1308/commits/d49b29cc34257f3a4e508c4449b308530eb455a8).